packages/react-reconciler/src/ReactFiberCompleteWork.js

```diff
@@ -1114,7 +1114,8 @@ function completeWork(
               return null;
             }
           } else if (
-            now() > renderState.tailExpiration &&
+            now() * 2 - renderState.renderingStartTime >
```

Comment:
Inline comment that translates this into English plz

"The time it took to render last row is greater than time until the expiration"